server.js

```diff
@@ -1,21 +1,20 @@
 const express = require("express");
+const cors = require("cors");
 const { backPort } = require("./conf");
-const passport = require("passport");
 
 const app = express();
 app.use(express.json());
-app.use(passport.initialize());
+app.use(cors());
 
-/* --------------------------------------------------------------------- Routes */
-
-app.use("/auth", require("./routes/auth"));
-
-/* --------------------------------------------------------------------- 404 and server launch */
-app.use((req, res) => {
-  const msg = `Page not found: ${req.url}`;
-  res.status(404).send(msg);
+app.get("/", (req, res) => {
+  console.log("GET / was called!");
+  res.send("Hi there!");
 });
```

Comment:
```suggestion
```
Maybe it's useless no ? And again a console.log ! 
Maybe you should remove this route